pkg/operator/staticresource.go

```diff
@@ -0,0 +1,171 @@
+package operator
```

Comment:
nit: we typically add these controller as separate packages, but I think it's fine like this for this operator.

Reply:
Moved to `staticresource/`